[2024-11-26T17:00:05.848+0900] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-26T17:00:05.858+0900] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_and_upload_json_to_hdfs.save_to_hdfs scheduled__2024-11-26T07:00:00+00:00 [queued]>
[2024-11-26T17:00:05.864+0900] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_and_upload_json_to_hdfs.save_to_hdfs scheduled__2024-11-26T07:00:00+00:00 [queued]>
[2024-11-26T17:00:05.864+0900] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-26T17:00:05.880+0900] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): save_to_hdfs> on 2024-11-26 07:00:00+00:00
[2024-11-26T17:00:05.883+0900] {standard_task_runner.py:72} INFO - Started process 3067083 to run task
[2024-11-26T17:00:05.886+0900] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'process_and_upload_json_to_hdfs', 'save_to_hdfs', 'scheduled__2024-11-26T07:00:00+00:00', '--job-id', '327', '--raw', '--subdir', 'DAGS_FOLDER/data_to_hdfs.py', '--cfg-path', '/tmp/tmp9kcd6d6r']
[2024-11-26T17:00:05.888+0900] {standard_task_runner.py:105} INFO - Job 327: Subtask save_to_hdfs
[2024-11-26T17:00:05.921+0900] {task_command.py:467} INFO - Running <TaskInstance: process_and_upload_json_to_hdfs.save_to_hdfs scheduled__2024-11-26T07:00:00+00:00 [running]> on host MN
[2024-11-26T17:00:05.994+0900] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='process_and_upload_json_to_hdfs' AIRFLOW_CTX_TASK_ID='save_to_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2024-11-26T07:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-26T07:00:00+00:00'
[2024-11-26T17:00:05.995+0900] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-26T17:00:06.008+0900] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-26T17:00:06.008+0900] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cat "None" | hdfs dfs -put -f - /data/match_data_combined.csv']
[2024-11-26T17:00:06.015+0900] {subprocess.py:99} INFO - Output:
[2024-11-26T17:00:06.016+0900] {subprocess.py:106} INFO - cat: None: No such file or directory
[2024-11-26T17:00:07.918+0900] {subprocess.py:110} INFO - Command exited with return code 0
[2024-11-26T17:00:07.939+0900] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-26T17:00:07.939+0900] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=process_and_upload_json_to_hdfs, task_id=save_to_hdfs, run_id=scheduled__2024-11-26T07:00:00+00:00, execution_date=20241126T070000, start_date=20241126T080005, end_date=20241126T080007
[2024-11-26T17:00:07.983+0900] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-26T17:00:08.001+0900] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-26T17:00:08.006+0900] {local_task_job_runner.py:245} INFO - ::endgroup::
